Sell/Basket/Models/SeatInfo.swift

```diff
@@ -33,10 +34,25 @@ public struct SeatInfo: Codable, Equatable {
 
 extension SeatInfo {
 
-    init(response: ApolloType.FragmentSeatInfo?) {
-        row = response?.row
-        seat = response?.seat
-        block = response?.block
-        table = response?.table
+    /// For GraphQL JSON type, when the response has no data for SeatInfo, we get `[]`, so we initialise SeatInfo with nil for this case.
+    /// On the other hand, if there's data for SeatInfo, we get [["row": "a", "seat": "b"]], we can initialise SeatInfo(row: "a", seat: "b")
```

Comment:
I'm confused about the response from BE. Is JSON a dictionary `[String: String]` or an array of dictionary `[[String: String]]`?